datasets/librispeech/_metadata.json

```diff
@@ -0,0 +1,254 @@
+            "contentUrl": "https://www.openslr.org/resources/12/intro-disclaimers.tar.gz",
+            "name": "intro-disclaimers.tar.gz",
+            "sha256": "92ba57a9611a70fd7d34b73249ae48cf",
+            "description": "[695M] extracted LibriVox announcements for some of the speakers",
+            "encodingFormat": "application/x-tar"
+        },
+        {
+            "@type": "FileObject",
+            "contentUrl": "https://www.openslr.org/resources/12/original-mp3.tar.gz",
+            "name": "original-mp3.tar.gz",
+            "sha256": "7e14b6df14f1c04a852a50ba5f7be915",
+            "description": "[87G] LibriVox mp3 files, from which corpus' audio was extracted",
+            "encodingFormat": "application/x-tar"
+        },
+        {
+            "@type": "FileObject",
+            "contentUrl": "https://www.openslr.org/resources/12/original-books.tar.gz",
+            "name": "original-books.tar.gz",
+            "sha256": "9da96b465573c8d1ee1d5ad3d01c08e3",
+            "description": "[297M] Project Gutenberg texts, against which the audio in the corpus was aligned",
+            "encodingFormat": "application/x-tar"
+        },
+        {
+            "@type": "FileObject",
+            "contentUrl": "https://www.openslr.org/resources/12/raw-metadata.tar.gz",
+            "name": "raw-metadata.tar.gz",
+            "sha256": "25eced105e10f4081585af89b8d27cd2",
+            "description": "[33M] Some extra meta-data produced during the creation of the corpus",
+            "encodingFormat": "application/x-tar"
+        },
+        {
+            "@type": "FileObject",
+            "contentUrl": "https://www.openslr.org/resources/12/md5sum.txt",
+            "name": "md5sum.txt",
+            "description": "[600 bytes] MD5 checksums for the archive files",
+            "encodingFormat": "text/plain"
+        },
+        {
+            "@type": "FileSet",
+            "containedIn": [
+                "#{dev-clean.tar.gz}",
+                "#{dev-other.tar.gz}",
+                "#{test-clean.tar.gz}",
+                "#{test-other.tar.gz}",
+                "#{train-clean-100.tar.gz}",
+                "#{train-clean-360.tar.gz}",
+                "#{train-other-500.tar.gz}"
+            ],
+            "name": "flacs-sources",
+            "includes": "*.flac",
+            "encodingFormat": "audio/x-flac"
+        },
+        {
+            "@type": "FileSet",
+            "containedIn": [
+                "#{dev-clean.tar.gz}",
+                "#{dev-other.tar.gz}",
+                "#{test-clean.tar.gz}",
+                "#{test-other.tar.gz}",
+                "#{train-clean-100.tar.gz}",
+                "#{train-clean-360.tar.gz}",
+                "#{train-other-500.tar.gz}"
+            ],
+            "name": "transcriptions-sources",
+            "includes": "*.trans.txt",
+            "encodingFormat": "text/plain"
+        }
+    ],
+    "recordSet": [
+        {
+            "name": "flacs",
```

Comment:
Maybe "audio-files" instead of "flacs", to make it easier to understand for people unfamiliar with the flac format?